docs/browser-compatibility.md

```diff
@@ -1,8 +1,10 @@
 # Browser compatibility
 
-The Reaction Commerce component library aims to support all browsers supported by React, v16, outlined [here](https://reactjs.org/docs/react-dom.html#browser-support).
+The Reaction Commerce component library aims to support all browsers supported by React v16, outlined [here](https://reactjs.org/docs/react-dom.html#browser-support).
 
 Browser list:
 - Google Chrome
+- Internet Explorer (11+)
```

Comment:
IE11 is the latest/last version of IE to be released. Edge followed and is a new browser (which is on the list already) 👍